web/app/test/ServiceMeshTest.jsx

```diff
@@ -50,7 +77,7 @@ describe('ServiceMesh', () => {
     });
   });
 
-  it("displays message for more than one deployment added to servicemesh", () => {
+  it("displays message for more than one deployment not added to servicemesh", () => {
```

Comment:
wording nit:
`displays a message if >1 deployment has not been added to the mesh`